arch/lkl/include/uapi/asm/swab.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _ASM_LKL_SWAB_H
```

Comment:
We also need to remove swab.h from Kbuild

Reply:
Does this diff (https://github.com/lkl/linux/pull/372/files#diff-051aff7b50396c2ae5a000bd891b5fc3) do the job you mentioned here ?

Reply:
okay, thanks.